glaring-client/src/components/Application.jsx

```diff
@@ -9,7 +9,7 @@ import Sidebar from "./Sidebar/Index";
 import { css, jsx } from "@emotion/core";
 
 export default function Application() {
-  const { state, setGroupData } = useAppData();
+  const { state, setGroupData, cancel } = useAppData();
 
   const { currentGroup, setCurrentGroup } = useState("Black Lives Matter");
```

Comment:
i'm going to mark every spot where this line is going to cause a merge conflict once either me or wares makes a pull request. there are five places where this is an issue.

you can do what you have to to for testing but _you need to clean up after yourself_ or i am going to reject your PRs. the function is already declared.

Reply:
You didn't do what I asked you to do. i asked you to clean up your test code and you didn't do it.

please don't do this to me